OpenShift.md

```diff
@@ -0,0 +1,55 @@
+# Tyk Helm Chart - OpenShift
```

Comment:
@zalbiraw @craigtyk I am not sure this belongs in this repo. Shouldn't it be in tyk.io/docs? We can put it there under FAQ since we are not ready to officially support it?

Reply:
We can move this to docs and link it.